README.md

```diff
@@ -19,30 +19,33 @@ via the Cython toolchain. Python in turn gives us the ability to work interactiv
 rich environment of tools for data analysis and scientific computing.
 Furthermore, NetworKit's core can be built and used as a native library if needed.
 
+# Install
 
-## Installation options
+There are three ways to install NetworKit as a Python module.
 
-We support three ways to install NetworKit:
+### Install with pip
 
--   NetworKit Virtual Machine: Download and try NetworKit preinstalled on a virtual machine. This is recommended for users using __Microsoft Windows__.
--   Pip install: Download the NetworKit Python package with pip. This is the easier way to get NetworKit but you can only use NetworKit via Python this way.
--   Build NetworKit from Source: Clone or download the source code of NetworKit and build the C++ and Python modules from source.
+    $ pip3 install [--user] networkit
 
-More detailed instructions follow after the requirements section.
-With NetworKit as a Python extension module, you get access to native high-performance code and can at the same time work interactively in the Python ecosystem. Although the standard Python interpreter works fine, we recommend [IPython] and [jupyterhub](https://github.com/jupyterhub/jupyterhub) as great environments for scientific computing.
+### Install with conda-forge
 
-Once you have installed NetworKit, please make sure to check out our [NetworKit UserGuide] for an overview of the features provided in NetworKit.
+    $ conda install networkit
```

Comment:
@fabratu should we add instruction on how to add the conda-forge channel?

Reply:
Yes - and for the concrete syntax I think we should go with the default readme from the feedstocks (without any extras):

conda config --add channels conda-forge

To exclude possible ambiguity we could also add:
conda install networkit -c conda-forge

Reply:
@fabratu Feel free to push a commit to this branch to fix this.